fixtures/0.8.1/valid.json

```diff
@@ -204,7 +204,7 @@
         "exp": 4804143412,
         "att": [
           {
-            "with": "prf/0",
+            "with": "prf:0",
```

Comment:
Would not this affect the actual token ? Or was this just a typo here ?

Reply:
As far as I can tell `token` field is also needs to be updated a per https://github.com/ipld/js-dag-ucan/runs/6458958822?check_suite_focus=true#step:5:107

Reply:
Pasting token into https://ucan.xyz/validator also confirmed my suspicion
<img width="1284" alt="image" src="https://user-images.githubusercontent.com/21236/168671332-87c36649-d536-408f-84cc-f3c348c4e200.png">

Reply:
Yeah, you're right. I don't know how to recreate the signature correctly with that issuer's key, so here's a different token that should do the same thing:
```
eyJhbGciOiJFZERTQSIsInR5cCI6IkpXVCIsInVjdiI6IjAuOC4xIn0.eyJhdWQiOiJkaWQ6a2V5Ono2TWtnWDVqalJVYnR5c2dnRTRyYUNhcUNYODhBelN2WXE4MVdKa0JvQTFvdDhhZSIsImF0dCI6W3sid2l0aCI6InByZjowIiwiY2FuIjoidWNhbi9ERUxFR0FURSJ9XSwiZXhwIjo0ODA0MTQzNDEyLCJmY3QiOltdLCJpc3MiOiJkaWQ6a2V5Ono2TWtoejNhdENZdGk5QktWUEc4alcyYlFCbnR5RW5ZS3BpTVJreTJuOGg0eVY1VSIsInByZiI6W119.TTqaclWZgPWDIjGk4tdHPHhzUE2NOLjGh7Wk3SX4GCQy9Kw3niD-9Q4ubTv2BoCk_anNEmCrmWUCijBd137pBg
```

I'll edit the PR.

Reply:
Actually, scrap that. The token I pasted above is incorrect, and I now see where the tokens are generated: https://github.com/ucan-wg/ucan-fixture-gen/blob/main/src/samples.ts